tests/e2e_pushdown/test_table_query_edge_cases.py

```
@@ -70,33 +76,35 @@ def test_quoted_identifiers_case_sensitive(single_source_env):
 
 
 def test_very_wide_select(single_source_env):
-    """Checks SELECT with many columns (wide projection) pushes correctly."""
+    """A wide projection of one repeated column dedups remotely, 20 cols locally.
+
+    The remote query fetches ``order_id`` once; the 20 aliases are produced by
+    the local projection, so the final result has all 20 columns.
+    """
     runtime = build_runtime(single_source_env)
     columns = ", ".join([f"order_id AS col{i}" for i in range(20)])
     sql = f"SELECT {columns} FROM duckdb_primary.main.orders"
```

Ruff (0.15.15)

[error] Possible SQL injection vector through string-based query construction

(S608)